### PR TITLE
[Enhancement] Changed hyperlinks in keynotes speakers and Zeomega sponsor

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Solves #152 :hatching_chick: 
And a small fix too in the silver sponsors.

### CODE REVIEW

File: templates/sections/_sponsors.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h3 class="wow fadeInDown">Silver Sponsors</h3>
       <ul class="tg-sponsors col-md-4">
         <li>
           <figure class="tg-sponder">
-             <a data-toggle="modal" href="#zeomega-modal" target="_blank" href="http://www.zeomega.com/">
+             <a href="http://www.zeomega.com/" target="_blank">
```
Comment (realslimshanky) [inline templates/sections/_sponsors.html]: Replace this with the previous code or `<a data-toggle="modal" href="#zeomega-modal">`

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please check out the above review.

File: templates/sections/_sponsors.html (hunk 1 above)
Comment (CuriousLearner) [reply templates/sections/_sponsors.html]: One task is to link all sponsors through modals as done with ZeOmega

File: templates/sections/_speakers.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h3 class = "wow fadeInDown text-center">Keynote Speakers</h3>
               </figcaption>
             </figure>
             <div class="tg-posttitle">
-              <h3><a href="">Peter Wang</a></h3>
+              <h3><a href="https://in.pycon.org/blog/2017/announcing-keynote-speakers-for-pycon-india-2017-wang.html">Peter Wang</a></h3>
               <span>CTO & Co-Founder at Continuum Analytics.</span>
```
Comment (CuriousLearner) [inline templates/sections/_speakers.html]: this should be `CTO &amp; Co-Founder`.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
The review is mentioned inline. Apart from that, rest is good.

File: templates/sections/_speakers.html (hunk 2 above)
Comment (vipulgupta2048) [reply templates/sections/_speakers.html]: Will correct it.

File: templates/sections/_sponsors.html (hunk 1 above)
Comment (vipulgupta2048) [reply templates/sections/_sponsors.html]: I didn't know that. I apologize. As by clicking on zeomega's link it didn't redirect to zeomega's homepage. Which I thought was it's real purpose. I will implement this.

Comment (vipulgupta2048) [reply templates/sections/_sponsors.html]: Replaced it with previous code

**Review by CuriousLearner — APPROVED** [review]

**Pull request merged**